src/languages/lambda/code.js

```diff
@@ -0,0 +1,349 @@
+"use strict";
+
+/* -----[ the parser ]----- */
+
+export default class Parser {
```

Comment:
Since this class only has static methods, there isn't really any need for it to be a class. I would just export the individual functions.